FFXIVClientStructs/FFXIV/Client/Graphics/Kernel/Texture.cs

```diff
@@ -48,8 +54,49 @@ public unsafe partial struct Texture {
     public partial void DecRef();
 }
 
-// there's 20+ of these but these are the ones I've encountered/debugged
+// See also DXGI_FORMATs that end with the same names.
 public enum TextureFormat : uint {
-    R8G8B8A8 = 5200,
-    D24S8 = 16976 // depth 28 stencil 8, see MS texture formats on google if you really care :)
+    B8G8R8A8_UNORM_4 = 0x1130,
```

Comment:
Probably named `L8_UNORM` but this is guessing based on what Lumina has documented